README.md

```diff
@@ -67,3 +67,27 @@ Prod:
 ```bash
 PORT=SOMEPORT npm start
 ```
+
+## Docker
+
+### Build docker image
+
+```bash
+docker build -f Dockerfile -t intermine/wizard:latest .
+```
+
+### Run wizard in a docker container
```

Comment:
Does this command have other dependencies?

Reply:
(e.g. containers running on some other port, etc.)

Reply:
Nope. Just docker

Reply:
And docker-compose for using docker-compose commands

Reply:
Hmm - when I run ```docker run --rm -p 9992:8080 intermine/wizard:latest ``` it tries to contact 127.0.0.1:9991.

Reply:
you can use docker-compose file to start wizard with all InterMine cloud dependencies in docker containers.

Reply:
right! But in that case we should explain that _this_ command expects other things to be running at certain locations, and to use docker-compose if you want to start them all at once, I think?

Reply:
Added a note about this in the README